_blog/2023-12-20-loco-rust-rails.mdx

```diff
@@ -213,7 +179,19 @@ ReportWorkerWorker::perform
 As you can see, we don't need to initialise the struct to use it - we can call the method from the struct directly and it will work assuming the arguments are valid - as you can see here, no arguments are required since the args struct does not have any fields.
 
 ## Deployment
-Currently, Loco.rs allows you to generate a deployment by using `cargo run generate deployment` - which lets you choose between Docker and Shuttle. When choosing Docker it will generate a Dockerfile that you can use to deploy anywhere, but when you pick Shuttle it will automatically generate everything you need for a Shuttle deployment - no further work required! You can then use the Shuttle CLI to start a new project and deploy it.
+Currently, Loco.rs allows you to generate a deployment by using the following comamnd:
+```bash
+cargo loco generate deployment
+```
+This lets you choose between Docker and Shuttle. When choosing Docker it will generate a Dockerfile that you can use to deploy anywhere, but when you pick Shuttle it will automatically generate everything you need for a Shuttle deployment - no further work required! You can then use the Shuttle CLI to start a new project and deploy it:
+```bash
+cargo shuttle start --name <name-of-project>
+cargo shuttle deploy
```

Comment:
If Shuttle.toml is not configured, the name flag is also needed on deploy, so add one of those.